### DISCUSSION TITLE
shen-ping/info/MQ9L6CofAKXneFibBxLzC

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#24] [Announcements]
# shen-ping/info/MQ9L6CofAKXneFibBxLzC

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/shen-ping/info/MQ9L6CofAKXneFibBxLzC?isInfo=false

**Comment by wurencaideli** [comment]
太真实了
